java/src/client/turntracker/TurnTrackerController.java

```diff
@@ -107,6 +110,7 @@ public void update(Observable observable, Object o) {
                 gameOver = true;
```

Comment:
Only comment I have is that the server is the one that decides the game is over. So none of the victory point logic needs to be here - IIRC the way the server indicates that the game is over is if `getModel().getWinner()` is set to not null - the server doesn't actually have a turn status for "game over".